### ISSUE TITLE
Munitions crates have wrong autochance value

### ISSUE THREAD

**Issue by certee** [kubikill/tf2unboxer#4]
Several munitions crates are not type 2 (everything has equal chance), they are type 0 (different items have different chances)

For this data, a few items are left out because they are obtainable from other sources

Munitions 85:
Strange Classic (strangifier) ~24000
Strange Manmelter (strangifier) ~24000
Strange Vaccinator (strangifier) ~3100
Strange Anger (strangifier) ~2000 (this includes strange angers unboxed directly from sniper crates, so its only slightly inflated)
Strange Apparition's Aspect (strangifier) ~4200 (this includes strange AA unboxed directly from pyro crates, so its only slightly inflated)

Munitions 92: (these are not strangifiers except for the last one)
Candy Cane: ~63000
Dalokohs Bar: ~74000
Warrior's Spirit: ~60000
Apoco-Fists: ~96000
Red-Tape Recorder: ~78000
Claidheamh Mòr: ~25000
Wrangler: ~27000
Shahanshah: ~23000
Cloak and Dagger: ~29000
Buff Banner: ~29000
Strange Rightous Bison (strangifier) ~9700

The actual numbers may be skewed due to a variety of factors such as not using the strangifiers or inventory scanners inability to see every item in existence, but it should be noted that there is a definite difference between the pull chances.  However, as the pull chances have never been released, an actual chance may be impossible to calculate.

**Comment by kubikill** [comment]
Thanks for the info! Could you please tell me where you got those item counts from? Some counts do not seem to match the counts from backpack.tf, like the Strange Vaccinator for example, as backpack.tf claims there are ~34000 Strange Vaccinators, including uncraftable versions.

The current item odds in unboxer.tf are based on the odds listed on the offical TF2 Wiki, but if the item counts you provided are indeed correct then I'll adjust the odds. It's unfortunate Valve has never released the actual odds of unboxing items in crates, but at least we can come close.

**Comment by certee** [comment]
Yes sorry, I did leave a zero off the vaccinator by accident. I was just getting ballpark figures so I didnt go into uncraftables. (figures should be Vacc ~33600, classic 29000, manmelter ~25000 with uncrafts, Widowmaker ~14500) 

I tried to find some mass unboxings but these crates never had that much of a draw. It should be noted there was a discussion on the  wiki on munition 92 that basically ended with them choosing to ignore the data and present these crates as evenly chanced https://wiki.teamfortress.com/wiki/Talk:Mann_Co._Supply_Crate#Crate_.2392_unbox_rate but they mustve reversed that line of thought and listed the chances for 92 on the wiki.

You had #92 correct already? Whoops. #85 might need adjusting, and #82 seems possibly wrong as well... 72000 strange phlogs versus about 6300 strange private eyes for example. If the strange parts were the same chances and the strangifiers, #82 would be a fantastic case to open. I can't find any mass box openings to verify. Tempted to throw down myself.... for science

**Comment by kubikill** [comment]
So while I was investigating, I found [this curious comment on TF2 Official Wiki](https://wiki.teamfortress.com/wiki/Talk:Mann_Co._Supply_Crate/Retired_series) about crate #82. It does appear that this crate has similar drop chances to #42, #49 and #77. While there isn't much data, the chances do appear to be going in that direction. I can see the same pattern for #85, too.

I'll tweak the chances of items in crates #82 and #85 to the following:

#82
Pretty Boy's Pocket Pistol Strangifier - 20%
Phlogistinator Strangifier - 20%
Cleaner's Carbine Strangifier - 20%
Strange Cosmetic Part: Kills - 10%
Strange Part: Full Health Kills - 10%
Strange Part: Headshot Kills - 10%
Private Eye Strangifier - 5%
Big Chief Strangifier - 5%

#85
Air Strike Strangifier - 15%
Classic Strangifier - 15%
Manmelter Strangifier - 15%
Vaccinator Strangifier - 15%
Strange Part: Robot Scouts Destroyed - 10%
Strange Part: Taunting Player Kills - 10%
Strange Part: Buildings Destroyed - 10%
Anger Strangifier - 3.33%
Apparition's Aspect Strangifier - 3.33%
Widowmaker Strangifier - 3.33%

I'll add this shortly after posting this comment. Thanks again for all the info :)
